yarn.lock

```diff
@@ -2,6 +2,23 @@
 # yarn lockfile v1
```

Comment:
It seems weird there were so many changes to `yarn.lock`.

Reply:
It seems there should have been zero changes to `yarn.lock`. In a rebase, I did a git-reset and dropped the changes to `yarn.lock`. Then I ran `yarn reboot:full` in the monorepo root and there were no changes to `yarn.lock`. A regenerated lock could introduce problems, so I recommend dropping the changes to `yarn.lock` in a rebase. Everything else looks good to go!